extensions/sts.md

```diff
@@ -36,9 +36,13 @@ Clients MUST ignore every token with a key that they don't understand.
 
 An STS policy has several parts:
 
-* An **upgrade policy**, expressed via the [`port` key](#the-port-key) over an insecure connection. *REQUIRED*
-* A **persistence policy**, expressed via the [`duration` key](#the-duration-key) over a secure connection. *REQUIRED*
-* A **preload policy**, expressed via the [`preload` key](#the-preload-key) over a secure connection. *OPTIONAL*
+* An **upgrade policy**, expressed via the [`port` key](#the-port-key). *REQUIRED* on an insecure connection.
+* A **persistence policy**, expressed via the [`duration` key](#the-duration-key) *REQUIRED* on a secure connection.
+* A **preload policy**, expressed via the [`preload` key](#the-preload-key) *OPTIONAL* on a secure connection.
```

Comment:
nitpick: "*OPTIONAL* on a secure connection." seems to imply that it isn't optional on an insecure connection, ie. that is required. The existing wording was less ambiguous for that item.

Reply:
I think that interpret would have to be a deliberately dishonest misinterpretation.